src/quizzes/chapter-4/4.0.1/service-ecosystems-quiz.js

```diff
@@ -0,0 +1,45 @@
+const rawQuizdown = `
+
+# You are working on your senior project and are designing a basic web app. Currently you are at the early stages of development and aren't hosting your project anywhere (ie you are using localhost to run)<br>What service ecosystem are you currently experiencing?
```

Comment:
```suggestion
# You are working on a new personal project and are designing a basic web app. Currently you are at the early stages of development and aren't hosting your project anywhere (ie you are using localhost to run)<br>What service ecosystem are you currently experiencing?
```

Remember this is open source and while geared towards our program not everyone is a senior in college :)